src/H5Fvfd_swmr.c

```diff
@@ -1886,3 +1958,46 @@ H5F_vfd_swmr_process_eot_queue(hbool_t entering_api)
 done:
     FUNC_LEAVE_NOAPI(ret_value)
 }
+
+/*-------------------------------------------------------------------------
+ *
+ * Function:    H5F_post_vfd_swmr_log_entry
+ *
+ * Purpose:     Write the log information to the log file.
+ *
+ * Parameters:
+ *              H5F_t *f                IN: HDF5 file pointer
+ *              int entry_type_code     IN: The entry type code to identify the
+ *                                          log entry tag.
+ *              char *log_info          IN: The information to be stored in the
+ *                                          log file.
+ * Return:   None
+ *
+ *-------------------------------------------------------------------------
+ */
+
+void
+H5F_post_vfd_swmr_log_entry(H5F_t *f, int entry_type_code, char *log_info)
+{
+    double        temp_time;
+    H5_timevals_t current_time;
+    char *        gettime_error;
+
+    /* Obtain the current time.
+       If   failed, write an error message to the log file.
+       else obtain the elapsed time in seconds since the log file
+            was created and write the time to the log file. */
+    if (H5_timer_get_times(f->shared->vfd_swmr_log_start_time, &current_time) < 0) {
+        gettime_error = HDmalloc(14);
```

Comment:
Error check for HDmalloc(14)?
Use define for 14 instead of a constant?

Reply:
Same reason. I don't know how HDF5 library is usually handled. If defining a constant is fine even if the logging is not used, I would like to do that.

Reply:
ok